leafmap/ai.py

```
@@ -0,0 +1,774 @@
+                traceback_text=result.get("traceback", ""),
+                description=description,
+                config=config,
+                system_prompt=system_prompt,
+                history=history,
+            )
+        except Exception as e:
+            if verbose:
+                print(f"[leafmap.ai] 修复调用失败: {e}")
+            break
+        if verbose:
+            print(f"[leafmap.ai] 修复后的代码:")
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)